src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -182,6 +182,12 @@ public class ApplicationParams {
     @Value("${enable-case-users-db-sync}")
     private boolean enableCaseUsersDbSync;
 
+    @Value("${ccd.jurisdiction-upload-timestamp-enabled}")
+    private boolean jurisdictionUploadTimestampEnabled;
+
+    @Value("${ccd.upload-timestamp-featured-jurisdiction}")
+    private String uploadTimestampFeaturedJurisdiction;
+
```

Comment:
The first one can be made redundant when we have a per-jurisdiction flag too. If we want it off for all then we just provide none for the list of jurisdictions. We'll also need to support multiple jurisdictions so something like the following is probably more suitable:
```suggestion
    @Value("#{'${ccd.upload-timestamp.enabled-jurisdictions}'.split(',')}")
    private List<String> uploadTimestampEnabledJurisdictions;
```